client/src/views/home.js

```diff
@@ -1,31 +1,130 @@
 /** @jsx jsx */
 import React from "react";
 import { jsx } from "@emotion/core";
-import { getClubs } from "../services/club";
-import { useClubs } from "../selectors/selectors";
 import { useSetClubs } from "../actions/action-hooks";
+import { useClubs } from "../selectors/selectors";
+import { getClubs } from "../services/club";
+import ClubCard from "../components/club-card";
 import Club from "../components/club";
+import { getDistance } from "geolib";
+import { Select } from "../components/ui";
 
 function Home() {
-  const [loading, setLoading] = React.useState(false);
   const clubs = useClubs();
   const setClubs = useSetClubs();
+  const [position, setPosition] = React.useState([0, 0]);
+  const [selectedLocation, setSelectedLocation] = React.useState();
+  const [sortType, setSortType] = React.useState("location");
 
   React.useEffect(() => {
-    setLoading(true);
     getClubs().then(clubs => {
       setClubs(clubs);
-      setLoading(false);
     });
-  }, []);
+  }, [setClubs]);
+
+  function setDistance(clubPosition, position) {
+    return getDistance(
+      { latitude: position[0], longitude: position[1] },
+      {
+        latitude: parseFloat(clubPosition.latitude),
+        longitude: parseFloat(clubPosition.longitude)
+      }
+    );
+  }
+
+  React.useEffect(() => {
+    const watchID = navigator.geolocation.watchPosition(pos => {
+      setPosition([pos.coords.latitude, pos.coords.longitude]);
+    });
+    return () => {
+      navigator.geolocation.clearWatch(watchID);
+    };
+  }, [setPosition]);
+
+  function handleChangeLocation(e) {
+    console.log(e.target.value);
+    setSelectedLocation(e.target.value);
+  }
+
+  function handleChangeSortType(e) {
+    console.log(e.target.value);
+    setSortType(e.target.value);
+  }
+
+  function sortBy(a, b) {
+    switch (sortType) {
+      case "location":
+        return a.distance - b.distance;
+
+      case "favorites":
+        return b.favorited_count - a.favorited_count;
+
+      case "name":
+        return a.name - b.name;
+
+      default:
+        break;
+    }
+  }
+
+  const styleSelectsContainer = {
+    display: "flex",
+    justifyContent: "space-between",
+    marginBottom: "1em"
+  };
 
   return (
     <div>
-      <h2>Filters</h2>
-      {loading && <div>Loading</div>}
-      {clubs.map(club => (
-        <Club club={club} key={club.id} />
-      ))}
+      <div css={styleSelectsContainer}>
+        <Select
+          onChange={handleChangeLocation}
+          defaultValue=""
```

Comment:
Could you assign a more significant name instead of `defaultValue` ?

Reply:
That's the default value for the select tag